cmd/gen-go-sample/sampmain.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"github.com/golang/protobuf/protoc-gen-go/descriptor"
+	"github.com/googleapis/gapic-generator-go/internal/pbinfo"
+	"strings"
+)
+
+func writeMain(g *generator, argNames []string, flagNames []string, argTrees []*initTree, methName string) error {
+	p := g.pt.Printf
+	st := newSymTab(nil)
+
+	// We only care about symbols here, not the types
+	dummyType := initType{}
+	for _, arg := range argNames {
+		if err := st.put(arg, dummyType); err != nil {
+			return err
+		}
+	}
+
+	p("func main() {")
+
+	for i := range argNames {
+		if _, ok := argTrees[i].typ.desc.(*descriptor.EnumDescriptorProto); ok {
+			p(`%s := flag.String(%s, "%s", "")`, argNames[i], flagNames[i], argTrees[i].leafVal)
```

Comment:
Does the first `%s` argument (`String(%s`) need to be wrapped in quotes? Or is it variable name? Just want to make sure because the following `p()` statement uses `%q` in the same position.

Reply:
Totally! Fixed.